README.md

```diff
@@ -114,4 +114,29 @@ To run `DefaultAzureCredentailAdapterTest`, ensure you have `.env` file created
 
 Once you have the `.env` file configured, run the test using JUnit 5 runner.
 
+## Python
+
+### default_azure_mgmt_credential.py
+
+The `DefaultAzureMgmtCredential` class provides a simple bridge to use `DefaultAzureCredential` from `Azure.Identity` SDKs. This is a convenient mechanism to authenticate all Azure Management Resources and a some data plane SDKs that use `ServiceClientCredential` family of credentials.
```

Comment:
Just this for this paragraph: The `DefaultAzureMgmtCredential` class provides a simple bridge to use `DefaultAzureCredential` from `Azure.Identity` SDKs with the Azure Management Plane SDKs.